src/Resources/config/commands.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <parameters>
+        <parameter key="leezy.pheanstalk.command.abstract.class">Leezy\PheanstalkBundle\Command\AbstractPheanstalkCommand</parameter>
```

Comment:
Hi @armetiz. 
What you think about adopting [this proposal](https://github.com/symfony/symfony/pull/12822) in ```"leezy/pheanstalk-bundle": "~3.0"```?